thoth/package_extract/image.py

```diff
@@ -525,6 +524,48 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    """
+    Sometimes this file can be not accessbile, in those cases we can grant access
+    # st = os.stat(path + "/usr/local/cuda/version.txt")
+    # os.chmod(path + "/usr/local/cuda/version.txt", st.st_mode | stat.S_IEXEC)
+    """
+    with open(path + "/usr/local/cuda/version.txt", "r") as f:
```

Comment:
@fridex It looks like this is the only way of getting the cuda version which is reading the file `version.txt`.

Reply:
Can you post an example of version.txt file?

Reply:
I pulled this image `nvidia/cuda` from docker hub and opened it with `sudo docker run -it nvidia/cuda bash`, then run `cat /usr/local/cuda/version.txt`:
```
CUDA Version 10.1.243
```

Reply:
Let's parse the file, so package-extract reports directly version identifier `10.1.243`. Don't make it dependent on relative line positioning.